helm/thingsboard/values.yaml

```diff
@@ -48,7 +48,7 @@ ingress:
 # endpoints in all services, refer to the Metrics
 # section in https://thingsboard.io/docs/user-guide/troubleshooting/
 monitoring:
-  enabled: false
```

Comment:
IMO we should not enable them by default.

Reply:
my fault, removed change